code/modules/antagonists/slaughter/slaughter.dm

```diff
@@ -39,13 +39,14 @@
 	/// How much our wound_bonus hitstreak bonus caps at (peak demonry)
 	var/wound_bonus_hitstreak_max = 12
 
-/mob/living/simple_animal/hostile/imp/slaughter/Initialize()
+/mob/living/simple_animal/hostile/imp/slaughter/Initialize(bloodpool)
```

Comment:
Is `bloodpool` being used as `loc` here?
Usually the first argument of `Initialize()` is `mapload`, which is not the same as `loc`